source/fundamentals/data-formats/document-data-format-json.txt

```diff
@@ -0,0 +1,45 @@
+==========================
+Document Data Format: JSON
+==========================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+
+Overview
+--------
+
+- Describe Extended JSON
+- Question here: what is the difference between toJson() and parse()?
+
+Read JSON
+---------
+
+- Description; mention BsonBinaryReader and JsonReader implementations
+- How to use parse/toJson to output for a BufferedReader
```

Comment:
toJson is used to write JSON, not read it

Reply:
Thanks, updated.